{"pytest_version": "7.2.2", "$report_type": "SessionStart"}
{"nodeid": "tests38_50_3.py::test_smaller_sets", "location": ["tests38_50_3.py", 6, "test_smaller_sets"], "keywords": {"test_smaller_sets": 1, "tests38_50_3.py": 1, "Folder_50": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00014417999773286283, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "2a603716d5cd414e96eec181e08de617", "node": "<WorkerController gw0>"}
{"nodeid": "tests38_50_3.py::test_largest_subset", "location": ["tests38_50_3.py", 18, "test_largest_subset"], "keywords": {"test_largest_subset": 1, "tests38_50_3.py": 1, "Folder_50": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.0002065170010610018, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "2a603716d5cd414e96eec181e08de617", "node": "<WorkerController gw1>"}
{"nodeid": "tests38_50_3.py::test_smaller_sets", "location": ["tests38_50_3.py", 6, "test_smaller_sets"], "keywords": {"test_smaller_sets": 1, "tests38_50_3.py": 1, "Folder_50": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/home/shahin/Benchmark_Command/Q38/command_results_3/Folder_50/generated_answer.py", "lineno": 2, "message": "AttributeError: 'set' object has no attribute 'count'"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_smaller_sets():", "        set_lengths = {i for i in range(1, 40)}", "        for i in set_lengths:", "            initial_set = {j for j in range(i)}", ">           assert find_subset_of_length_n(initial_set) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q38/command_results_3/Folder_50/tests38_50_3.py", "lineno": 11, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def find_subset_of_length_n(set_of_elements):", ">       return 1 << set_of_elements.count(1)", "E       AttributeError: 'set' object has no attribute 'count'"], "reprfuncargs": {"args": [["set_of_elements", "{0}"]]}, "reprlocals": null, "reprfileloc": {"path": "Q38/command_results_3/Folder_50/generated_answer.py", "lineno": 2, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_smaller_sets():", "        set_lengths = {i for i in range(1, 40)}", "        for i in set_lengths:", "            initial_set = {j for j in range(i)}", ">           assert find_subset_of_length_n(initial_set) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q38/command_results_3/Folder_50/tests38_50_3.py", "lineno": 11, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    def find_subset_of_length_n(set_of_elements):", ">       return 1 << set_of_elements.count(1)", "E       AttributeError: 'set' object has no attribute 'count'"], "reprfuncargs": {"args": [["set_of_elements", "{0}"]]}, "reprlocals": null, "reprfileloc": {"path": "Q38/command_results_3/Folder_50/generated_answer.py", "lineno": 2, "message": "AttributeError"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/home/shahin/Benchmark_Command/Q38/command_results_3/Folder_50/generated_answer.py", "lineno": 2, "message": "AttributeError: 'set' object has no attribute 'count'"}, null]]}, "when": "call", "user_properties": [], "sections": [], "duration": 0.00012857100227847695, "extra": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "2a603716d5cd414e96eec181e08de617", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
